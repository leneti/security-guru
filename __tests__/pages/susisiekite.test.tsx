import { render } from "@testing-library/react";
import ContactUs from "@site/pages/susisiekite";

jest.mock("@site/components/PageBackground");
jest.mock("@site/components/GetInTouch");

describe("Contact-us page", () => {
  it("renders", () => {
    const { container } = render(<ContactUs />);
    expect(container).toMatchSnapshot();
  });
});
