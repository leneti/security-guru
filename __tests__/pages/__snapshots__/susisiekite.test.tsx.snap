// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact-us page renders 1`] = `
<div>
  <div
    data-testid="Mock-PageBackground Susisiekite"
  >
    <div>
      Mock GetInTouch
    </div>
  </div>
</div>
`;
